src/layouts/BaseLayout.astro

```diff
@@ -105,6 +105,7 @@ const description = `JSHeroes is an yearly event organized by the local JS commu
         background-color: var(--color-neutral-light);
         line-height: 1.6;
         overscroll-behavior-y: none;
+        scroll-behavior: smooth;
```

Comment:
I wouldn't use this globally, it is a very strong animation for people with motion sensitivity. maybe add it for this particular page only. on the main page it scrolls a lot of content to go through the agenda for example, and it can be quite annoying. also, on the media page it could be added behind a `prefers-reduced-motion` media query